src/applications/edu-benefits/tests/1990/01-all-fields.e2e.spec.js

```diff
@@ -185,11 +185,12 @@ const test = E2eHelpers.createE2eTest(client => {
     .element('.js-test-location')
     .attribute('data-location')
     .to.not.contain('/review-and-submit')
-    .before(Timeouts.submission);
+    .before(Timeouts.slow);
 
   // Confirmation page
   client.waitForElementVisible('.confirmation-page-title', Timeouts.normal);
   client.axeCheck('.main').end();
 });
 
 module.exports = test;
+module.exports['@tags'] = ['integration'];
```

Comment:
This is how you enable a test to run in the integration test set that runs them against staging.